### DISCUSSION TITLE
Retrying failed queries

### DISCUSSION THREAD

**Discussion by gunar** [hasura/graphql-engine#6169] [General]
What would Hasura do if a single query fails?

> Aurora serverless, during scaling events the backend can lose a query or 2 (aws tries to find a good time to scale) and I have to account for those transactions once the event has finished. I was just curious if Hasura had any way to retry those queries that aren't successfully resolved ([source](https://hasura.io/blog/aws-aurora-is-it-for-you/#comment-5144575191))

**Comment by tirumaraiselvan** [comment]
GraphQL queries will throw a user-facing error and these could be retried from the client.
